build/flutter_app.gni

```diff
@@ -105,7 +104,7 @@ template("flutter_app") {
       "--output-file",
       rebase_path(bundle_path),
       "--snapshot",
-      rebase_path(snapshot_path),
+      rebase_path(snapshot_path, root_build_dir),
```

Comment:
Why does this path need to be rebased to the root_build_dir but not all the other paths?